src/shogun/multiclass/tree/KDTree.h

```diff
@@ -40,7 +40,7 @@ namespace shogun
 {
 
 /** @brief This class implements KD-Tree.
- * cf. http://www.autonlab.org/autonweb/14665/version/2/part/5/data/moore-tutorial.pdf
+ * cf. https://web.archive.org/web/20140327060654if_/http://www.autonlab.org:80/autonweb/14665/version/2/part/5/data/moore-tutorial.pdf
```

Comment:
is this really a good idea? Why not put a reference as a citation (a different/better reference, or wikipedia)

Reply:
I am not the author of code, so I decided to put what the author meant when was writing this comment

Reply:
I saw that link from the archive works, I do not really investigate content. Sorry, will do better next time.

Reply:
no worries at all. I just think dead-links should be removed rather than pulled out from the internet archive ;) wikepedia is probably fine here anyways.